libethereum/Executive.cpp

```diff
@@ -180,6 +180,8 @@ void Executive::initialize(Transaction const& _transaction)
 {
 	m_t = _transaction;
 
+	m_sealEngine.verifyTransaction(ImportRequirements::Everything, _transaction, m_envInfo);
```

Comment:
@winsvega We still need to wrap this in try/catch, see my comment in https://github.com/ethereum/cpp-ethereum/pull/3655